json/pt.json

```diff
@@ -1,65 +1,151 @@
     "Not Found": "Não encontrado",
     "Oh no": "Oh não",
+    "Once a team is deleted, all of its resources and data will be permanently deleted. Before deleting this team, please download any data or information regarding this team that you wish to retain.": "Uma vez eliminada a equipa, todos os seus recursos e dados serão eliminados de forma permanente. Antes de eliminar esta equipa, por favor descarregue todos os dados ou informações relativos a esta equipa que deseje manter.",
     "Page Expired": "Página expirou",
     "Password": "Palavra-passe",
+    "Permanently delete this team.": "Eliminar esta equipa permanentemente.",
+    "Permanently delete your account.": "Eliminar a sua conta permanentemente.",
+    "Permissions": "Permissões",
+    "Photo": "Foto",
     "Please click the button below to verify your email address.": "Por favor, clique no botão em baixo para verificar seu endereço de e-mail.",
-    "Please confirm access to your account by entering one of your emergency recovery codes.": "Please confirm access to your account by entering one of your emergency recovery codes.",
-    "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Please confirm access to your account by entering the authentication code provided by your authenticator application.",
+    "Please confirm access to your account by entering one of your emergency recovery codes.": "Por favor confirme o acesso à sua conta inserindo um dos seus códigos de recuperação de emergência.",
+    "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Por favor confirme o acesso à sua conta inserindo o código fornecido pela a sua aplicação de autenticação.",
     "Please confirm your password before continuing.": "Por favor, confirme sua palavra-passe antes de continuar.",
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie o seu novo API Token. Por razões de segurança, não será mostrado novamente.",
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor insira a sua palavra-passe para confirmar que pretende terminar as suas outras sessões em browser em todos os seus dispositivos.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor forneça o endereço de e-mail da pessoa que pretende adicionar a esta equipa. O endereço de e-mail deve estar associado a uma conta que já existe.",
+    "Profile Information": "Informação de Perfil",
+    "Profile": "Perfil",
+    "Recovery Code": "Código de Recuperação",
+    "Regenerate Recovery Codes": "Regenerar Códigos de Recuperação",
     "Regards": "Atenciosamente",
     "Register": "Registar",
     "Remember Me": "Lembrar-me",
```

Comment:
Lembrar-Me ? To be coherent with "Remember Me"

Reply:
I know it's a question for all translations

Reply:
Not in this case: 
1. 1.  "Remember Me" and "Remember me" are both in JSON file;
2. in pt you do not use the uppercase unless, by example, related to divinity like "And then God said: Remember-Me".

And by the way i would like to thank you for the work you have done.

Cheers!